app/assets/javascripts/views/project_search_view.js

```diff
@@ -1,9 +1,11 @@
+import Operands from '../mixins/contextual_serach_operands';
```

Comment:
Prefer to create a React component for that instead of a Backbone view, we're migrate away from Backebone views https://github.com/Codeminer42/cm42-central/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20react

Reply:
That backbone view were already part of the system. As for that that import, it is just a helper for the aliasing of some operands.
I shall start updating it to React after this task.